src/Github.hs

```diff
@@ -127,13 +142,16 @@ instance FromJSON PullRequestPayload where
 
 instance FromJSON CommentPayload where
   parseJSON (Object v) = CommentPayload
-    <$> (v .: "action")
+    <$> ((Left <$> v .: "action") <|> (Right <$> v .: "action"))
```

Comment:
I think `edited` will now always parse as `CommentEdited`. In this case it does not matter for the logic because we don't handle the edited event anyway, but if we do want to handle it in the future, it will sure be confusing for the person debugging, why the review edit events are not coming through. Is there some other key that we can use to distinguish? And then do something like

```haskell
do
  action <- case ? of
    "issue_comment" -> Left <$> v .: "action"
    "pull_request_review" -> Right <$> v .: "action"
  pure $ CommentPayload action <$> ...

Reply:
Good catch!